### ISSUE TITLE
<select> opens outside window on first try

### ISSUE THREAD

**Issue by mq1** [tauri-apps/tauri#1911]
`<select>` elements open outside the application window. When opened a second time, they open in the correct position. The problem occurs again when moving the window to a new location.

OS: Windows, version 10.0.19043 X64
Node: 15.14.0
Rustc: 1.52.1

**Comment by lucasfernog** [comment]
Can you provide a HTML example to reproduce it?

**Comment by mq1** [comment]
> 
> 
> Can you provide a HTML example to reproduce it?

I created this repo https://github.com/mq1/tauri-issue

**Comment by lucasfernog** [comment]
The issue doesn't happen on my end. What's the output of `tauri info`? I'd try updating the webview runtime to latest. Mine is `Webview2 - 91.0.864.41`.

**Comment by lucasfernog** [comment]
I'm closing this issue since it seems like an old Windows webview issue. Please reopen if you find something else.

**Comment by mq1** [comment]
I reopened the issue (https://github.com/tauri-apps/tauri/issues/1953)
My Webview2 version is the same as yours

**Comment by mq1** [comment]
Maybe we'll find other people with the same issue

**Comment by lucasfernog** [comment]
Also, do you see this issue when opening the HTML on the chromium Edge?

**Comment by mq1** [comment]
> 
> 
> Also, do you see this issue when opening the HTML on the chromium Edge?

No, it works correctly

**Comment by BenoitRanque** [comment]
I was able to reproduce this issue on windows.

**Comment by euphbriggs** [comment]
Is this related to DPI scaling?

Are the folks who experience this using a DPI scale factor other than 100% in their Windows display settings?

**Comment by lucasfernog** [comment]
I can reproduce this on my laptop with stock configuration.

**Comment by euphbriggs** [comment]
Is there any chance your machine defaults to a scale factor other than 100%?

**Comment by lucasfernog** [comment]
It's 100%. Seems like the webview isn't recognizing the window position change.

**Comment by euphbriggs** [comment]
Oh ok, thanks for checking.

**Comment by lucasfernog** [comment]
@amrbashir just posted this on Discord:

https://github.com/MicrosoftEdge/WebView2Feedback/issues/780
https://github.com/MicrosoftEdge/WebView2Feedback/issues/780#issuecomment-808306938

**Comment by nicholasdgoodman** [comment]
I have just put a code sample of a hacky but functioning workaround for this issue over on https://github.com/MicrosoftEdge/WebView2Feedback/issues/780 if you are looking for a quick fix.

*tl;dr*

If you "wiggle" the bounds of the WebView2 it will correctly recalibrate its dropdown positions.

**Comment by lucasfernog** [comment]
Thanks for the help @nicholasdgoodman I'll patch this fix in our webview library.

**Comment by amrbashir** [comment]
Re-opening this, since we reverted the workaround, because it introduced more bugs than it solved.

**Comment by FabianLars** [comment]
This was fixed in webview2 directly a few months ago. There is still an issue with _opened_ dialogs not moving with the window.
